### DISCUSSION TITLE
Set glass color seperate from accent color

### DISCUSSION THREAD

**Discussion by Sand216** [ALTaleX531/OpenGlass#76] [Q&A] [answered]
When using `ForceAccentColorization`, OpenGlass reads from `AccentColor` and `AccentColorInactive` to get colors. You can modify `AccentColor` via the registry to force a different color, but it gets reset to the Windows accent color after theme switch/reboot. `AccentColorInactive` isn't used by Windows as far as I know so it doesn't get reset.

I want to set a different color for glass than my accent color, is there a way to do this? Maybe an `AccentColorOverride` key would have to be added?

**Comment by ALTaleX531** [comment]
No, you can't. OpenGlass's glass color is always bound to the system's accent color or the color actually displayed in the title bar. I am not sure exactly what kind of settings you want.

**Reply by Sand216** [reply]
> I am not sure exactly what kind of settings you want.

It would be nice to have a way to set glass colors independently from the selected accent color. In niche scenarios, the color you want to use as the glass color does not work well as an accent color. Since accent colors have a set range of lightness and darkness when set through settings, you need to use a theme file to set the accent color to be as accurate as possible.

**Reply by ALTaleX531** [reply, marked as answer]
I may implement similar mitigations in the future, but I still believe this limitation of Windows is perfectly reasonable, and that your request is better satisfied to be met in DWMBlurGlass.

**Reply by Sand216** [reply]
I understand, thank you for considering it. I recently migrated from DWMBlurGlass due to its instability, and I was not sure if this would be within scope.
